terraform/prod.images.tfvars

```diff
@@ -1,2 +0,0 @@
-webapp_image_tag  = "ac45d5ecf8650d765bebc642e1dd1dcc48e0229d"
```

Comment:
Did you mean to remove this file?

Reply:
Yes, sorry should have commented.  I had some bugs where I was using "prod" in some places but "production" in others.  Decided to be consistent and use "production" universally, so removed this file and replaced with `production.images.tfvars`.